scipy/stats/_stats_py.py

```diff
@@ -5125,6 +5156,7 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate',
 
     Examples
     --------
+    >>> import numpy as np
     >>> from scipy import stats
```

Comment:
Not needed in this example
```suggestion
    --------
    >>> from scipy import stats
```